docs/usage/webhook/deploy.rst

```diff
@@ -0,0 +1,64 @@
+.. _usage_ffmpeg_deploy:
+
+Deploying on http server
+========================
+
+Dataflow and Config files
+-------------------------
+
+Create a ``Dataflow`` with operation ``convert_to_gif`` and save it in ``deploy/df/ffmpeg.yaml``
+
+.. code-block:: console
+
+    $ mkdir -p deploy/mc/http deploy/df
+    $ cat > /tmp/operations <<EOF
+    convert_to_gif
+    EOF
+
+    $ dffml dataflow create -config yaml $(cat /tmp/operations) > deploy/df/ffmpeg.yaml
+
+Create the `Config <../../plugins/service/http/dataflow.html#HttpChannelConfig>`__ file for the http server
+in ``deploy/mc/http/ffmpeg.yaml``
+
+.. code-block:: console
+
+    $ cat > ./deploy/mc/http/ffmpeg.yaml <<EOF
+    path: /ffmpeg
+    presentation: json
```

Comment:
Remember that we want the output to be the bytes of the `output.gif` file written directly to the client (preferably with a `Content-Type` header set to whatever is specified in `ffmpeg.yaml` as `content_type`). And we want to take the post body and treat it as the input file. Ideally we'd have it as `stream` and pass that stream as the stdin to the call to subprocess. However, we may have write those bytes out to a temporary file, and then pass the temporary file name into the ffmpeg operation.

Reply:
@pdxjohnny by `content_type` do you mean,`presentation`,or should i add another param?

Reply:
also when you say we write the output file back to the client,we'd have to have an option to do
```
return web.Response(body = result,content_type = config.content_type)
```
in place of 
https://github.com/aghinsa/dffml/blob/bbfec63078863247b2bba4ff6ce60e7403cce4d7/service/http/dffml_service_http/routes.py#L267-L281

is that the idea?
if we are doing that,ig it makes sense to do [issue#608](https://github.com/intel/dffml/issues/608) along with this.

Reply:
Yes, and yes we should do #608 along with this.